Formula/gnuradio.rb

```diff
@@ -1,10 +1,11 @@
 class Gnuradio < Formula
+  include Language::Python::Virtualenv
+
   desc "SDK providing the signal processing runtime and processing blocks"
   homepage "https://gnuradio.org/"
-  url "https://gnuradio.org/releases/gnuradio/gnuradio-3.7.13.4.tar.gz"
-  sha256 "c536c268b1e9c24f1206bbc881a5819ac46e662f4e8beaded6f3f441d3502f0d"
+  url "https://github.com/gnuradio/gnuradio/releases/download/v3.8.1.0/gnuradio-3.8.1.0.tar.gz"
```

Comment:
Why are we not keeping their website?

Reply:
Looks like they've stopped uploading new releases to their website.

Reply:
Related: gnuradio/gnuradio#3672

Reply:
> @SeekingMeaning Thanks for pointing that out. We'll get the release artifacts copied over to there.

Will re-resolve this once updated